Maple2Storage/Types/InventoryType.cs

```diff
@@ -1,5 +1,5 @@
 ﻿namespace Maple2Storage.Types {
-    public enum InventoryTab : byte {
+    public enum InventoryType : byte {
```

Comment:
Why was this changed?

Reply:
I think we should just keep the old one. It's pretty self explanatory what it is and it's already used in many files. If this needs to be changed, I'd say we should make another PR that only has the refactoring changes.